API.md

```diff
@@ -422,12 +423,28 @@ rename :: String -> String -> { k: v } -> { k: v }
 
 Easily rename a property on an object to be a different key.
 
-See also [`copyProp`](#copyprop).
+See also [`copyProp`](#copyprop), [`renameAll`](#renameall).
 
 ```js
 rename('id', 'courseId', { id: 'abc' }) //=> { courseId: 'abc' }
 ```
 
+### renameAll
+
+`@articulate/funky/lib/renameAll`
+
+```haskell
+renameAll :: { k: v } -> { k: v } -> { k: v }
+```
+
+Rename multiple properties on an object to have different keys.
+
+```js
+const orig = { first_name: 'Miles', last_name: 'Callisto' }
+renameAll({ first_name: 'firstName', last_name: 'lastName' }, orig)
+//=> { firstName: 'Miles', lastName: 'Callisto' }
```

Comment:
Maybe I'm missing something, but this seems backwards to me.
```js
const deIdentify =
  renameAll({ first_name: 'firstName', last_name: 'lastName' })

deIdentify({ first_name: 'Miles', last_name: 'Callisto' })
```
says to me that the result should be
```js
//=> { firstName: 'firstName', lastName: 'lastName' }
```
I believe the first argument should be what the outcome should be, then the last one is the `orig`inal data.

Reply:
I'm so sorry; I'm still a little confused. The pseudo-type looks like this currently, right?
```haskell
OriginalObject -> RenameObject -> NewObject
```
whereas I'm asking if it should be:
```haskell
RenameObject -> OriginalObject  -> NewObject
```
So when you mention `{ frum: to }` above, I'm not quite sure what we're talking about, unfortunately. I'm also not as deep into this project, so I may have no clue what I'm talking about.

With what's currently here, if I want to compose, I'll need to do this?

```js
const deIdentify =
  renameAll(__, { first_name: 'firstName', last_name: 'lastName' })

const doStuff =
  compose(deIdentity, somethingElse)

doStuff({ first_name: 'Miles', last_name: 'Callisto', email: 'mc@mc.org' })
//=> { firstName: 'firstName', lastName: 'lastName', email: 'mc@mc.org' }
```

Reply:
Yeah, now I'm confused.  `renameAll` already takes the renames first and the original second:
https://github.com/articulate/funky/blob/a02064529ca637f5289b78643e3750fccd13e959/src/renameAll.js#L4

Remember, we're renaming **keys**, not **values**.  Other existing functions like `R.merge` work on the values.

Reply:
ffs. what is wrong with me

Reply:
ship it